[DOCS] Instance ID field (PMM-13157) (#4434)
percona/pmm: documentation/docs/release-notes/3.4.0.md

For configuration options and examples, see [Configure environment variables for PMM Server](../install-pmm/install-pmm-server/deployment-options/docker/env_var.md#configure-vmagent-on-pmm-client).

### Enhanced management of RDS instances

PMM now supports tracking RDS Instance IDs when adding Amazon RDS databases to monitoring, making it easier to correlate instances between PMM and the AWS console.

> When adding RDS instances via **Inventory > Add Service**, you can now specify an Instance ID that matches your AWS RDS instance identifier. This optional field helps you manage large RDS deployments by providing a clear reference between monitored instances in PMM and their corresponding resources in AWS. 

Reviewer: ```suggestion
When adding RDS instances via **Inventory > Add Service**, the Instance ID that you specify will now be persisted. This optional field helps you manage large RDS deployments by providing a clear reference between monitored instances in PMM and their corresponding resources in AWS. 
```

Reviewer: FYI: it wasn't persisted before this change.
